pinot-minion/src/main/java/org/apache/pinot/minion/executor/MinionTaskZkMetadataManager.java

```diff
@@ -31,9 +31,11 @@
  */
 public class MinionTaskZkMetadataManager {
   private final HelixManager _helixManager;
+  private final int _endReplaceSegmentsSocketTimeoutMs;
 
-  public MinionTaskZkMetadataManager(HelixManager helixManager) {
+  public MinionTaskZkMetadataManager(HelixManager helixManager, int endReplaceSegmentsSocketTimeoutMs) {
```

Comment:
+1 that we should not change the interface. However, adding this config to `MinionTaskZkMetadataManager` looks like a hacky solution to me. 
Here's the class definition.
```
 * An abstraction on top of {@link HelixManager}, created for the {@link PinotTaskExecutor}, restricted to only
 * get/update minion task metadata
``` 

Let's brainstorm the better way (e.g. add the new method instead of changing the interface)